BMP.md

```diff
@@ -0,0 +1,64 @@
+# BGP Monitoring Protocol (BMP)
+
+https://tools.ietf.org/html/rfc7854
+
+## BMP message types and GoBMP:
+GoBMP will not need to collect and parse the following BMP message types:
```

Comment:
In both statements you say GoBMP will **not**, I suspect one of these should be without "not" right?

Reply:
fixed, thanks